pkg/firewall/firewall.go

```diff
@@ -70,41 +68,38 @@ type stakeOrActiveKeepPolicy struct {
 func (soakp *stakeOrActiveKeepPolicy) Validate(
 	remotePeerPublicKey *ecdsa.PublicKey,
 ) error {
-	remotePeerNetworkPublicKey := coreKey.NetworkPublic(*remotePeerPublicKey)
-	remotePeerAddress := coreKey.NetworkPubKeyToChainAddress(
-		&remotePeerNetworkPublicKey,
-	)
+	remotePeerOperatorID := soakp.chain.PublicKeyToOperatorID(remotePeerPublicKey)
```

Comment:
Yesssss! Removing this insane `coreKey.NetworkPublic` -> `coreKey.NetworkPubKeyToChainAddress` conversion warms my heart. :fire: ❤️